app/build.gradle

```diff
@@ -62,4 +72,17 @@ dependencies {
     // Facebook SDK
     implementation "com.facebook.android:facebook-login:$rootProject.ext.facebookLoginVersion"
 
-}
\ No newline at end of file
+    // Firebase
+    implementation "com.google.firebase:firebase-database:$rootProject.ext.playServicesVersion"
+    implementation "com.google.firebase:firebase-core:$rootProject.ext.playServicesVersion"
+    //Firebase SDK
+    compile "com.google.firebase:firebase-auth:$rootProject.ext.playServicesVersion"
```

Comment:
If you add these, please remove the ones i added above.

Reply:
Ok

Reply:
In fact it not the same library